docs/flocker-features/emc-coprhd-configuration.rst

```diff
@@ -0,0 +1,14 @@
+.. _emc-coprhd-backend:
+
+=================================
+ EMC CoprHD Block Device Backend
```

Comment:
remove "EMC"  per driver developer:

When you publish in community supported drivers, CoprHD may *not* come with in EMC Block Device Backend. It can be mentioned separate as “CoprHD Software Defined Storage”.